polygeist/tools/cgeist/Test/Verification/sycl/kernels.cpp

```diff
@@ -47,10 +49,6 @@ void host_1() {
   }
 }
 
-// CHECK: func.func @_ZTSZZ6host_2vENKUlRN4sycl3_V17handlerEE_clES2_E8kernel_2(%arg0: memref<?xi32>, %arg1: !sycl_range_1_, %arg2: !sycl_range_1_, %arg3: !sycl_id_1_)
```

Comment:
I don't understand why this is no longer generated.

Reply:
Guessing he wants to check it only in `kernels_funcs.cpp`.

Reply:
As per the last commit, ([see](polygeist/tools/cgeist/driver.cc:989)), if the input contains a kernel, only the device module is generated.

Reply:
No prob. this is what code reviews are for.